plugins/generator-1.19.4/forge-1.19.4/mappings/mobspawntypes.yaml

```diff
@@ -0,0 +1,9 @@
+_default: MobCategory.MONSTER
+ambient: MobCategory.AMBIENT
+axolotls: MobCategory.AXOLOTLS
+creature: MobCategory.CREATURE
+misc: MobCategory.MISC
+monster: MobCategory.MONSTER

```

Comment:
Monster is the first one looking at vanilla code, so this list is not in the order yet